### PR TITLE
feat: prerendering for landing page

### PR DESCRIPTION
**Pull request by preschian**
- [x] closes https://github.com/kodadot/nft-gallery/issues/6308

tested on https://www.opengraph.xyz/url/https%3A%2F%2Fkodadot.preschian.xyz%2F

### CODE REVIEW

File: ssr-opengraph/src/index.tsx [hunk 1]
```diff
@@ -6,12 +6,28 @@ import { collectionDetail, galleryDetail, userDetail } from './handlers';
 
 const app = new Hono();
 
-app.get('/', (c) => {
-  return c.text('hello hono.js');
-});
-
 const chains = ['bsx', 'snek', 'rmrk', 'ksm'];
```
Comment (roiLeo) [inline ssr-opengraph/src/index.tsx:9]:
```suggestion
const chains = ['bsx', 'snek', 'rmrk', 'ksm', 'stmn'];
```
?

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

File: ssr-opengraph/src/index.tsx [hunk 2]
```diff
@@ -6,11 +6,27 @@ import { collectionDetail, galleryDetail, userDetail } from './handlers';
...
+  }
+
+  const props = {
+    name: 'KodaDot',
+    siteData: {
+      title: 'KodaDot - NFT Market Explorer',
+      description: 'One Stop NFT Shop on Polkadot',
+      canonical: 'https://kodadot.xyz',
+      image: 'https://kodadot.xyz/k_card.png',
+    }
```
Comment (roiLeo) [inline ssr-opengraph/src/index.tsx:25]: So if I understand the one set on `nuxt.config.js` cannot be taken into account?

Comment (preschian) [reply ssr-opengraph/src/index.tsx:25]: correct, what we change in our `nuxt.config.js` won't reflect in our prerendering case for now

File: ssr-opengraph/src/index.tsx [hunk 3]
```diff
@@ -6,11 +6,27 @@ import { collectionDetail, galleryDetail, userDetail } from './handlers';
...
+  const useragent = c.req.headers.get('user-agent');
+
+  if (useragent && !isbot(useragent)) {
+    return fetch(c.req.url);
+  }
+
+  const props = {
+    name: 'KodaDot',
+    siteData: {
+      title: 'KodaDot - NFT Market Explorer',
```
Comment (roiLeo) [inline ssr-opengraph/src/index.tsx:21]:
```suggestion
      title: 'KodaDot - Go To Polkadot NFT marketplace',
```

File: ssr-opengraph/src/index.tsx [hunk 4]
```diff
@@ -6,11 +6,27 @@ import { collectionDetail, galleryDetail, userDetail } from './handlers';
...
+      description: 'One Stop NFT Shop on Polkadot',
```
Comment (roiLeo) [inline ssr-opengraph/src/index.tsx:22]:
```suggestion
      description: 'Multi-chain NFT marketplace across Polkadot parachains',
```

**Review by roiLeo — COMMENTED** [review]
and we are good

**Pull request merged**